Documentation/op-guide/maintenance.md

```diff
@@ -62,7 +62,7 @@ The space quota can be triggered with a loop:
 
 ```sh
 # fill keyspace
-$ while [ 1 ]; do dd if=/dev/urandom bs=1024 count=1024  | etcdctl put key  || break; done
+$ while [ 1 ]; do dd if=/dev/urandom bs=1024 count=2  | etcdctl put key  || break; done
```

Comment:
With a small quota value the original code (1 MiB puts) won't fill up the keyspace since it will hit the quota alarm.  2 KiB should always work.

Also, I just noticed this should be ```etcdctl set key```.